publisher-sdk/src/main/java/com/criteo/publisher/Criteo.java

```diff
@@ -216,4 +225,6 @@ public void setMopubConsent(@Nullable String ignored) {
 
   public abstract void setUserData(@NonNull UserData userData);
 
+  public abstract void setTagForChildDirectedTreatment(@Nullable Boolean flag);
```

Comment:
The wording is quite weird:
* the method is named: `setTagFor...`
* the parameter is named: `flag`

Hence you have 2 different words on the same declaration/meaning. Please use a single one.
Also, I believe that the name `tag for child directed treatment` is a Google word. Do we want to follow Google wording ? Or do we want to use something more generally used in the industry ? For instance:
* IAB: https://www.iab.com/wp-content/uploads/2019/10/IAB_2019-10-09_Navigating-COPPA-Guide.pdf
* Meta: https://developers.facebook.com/docs/audience-network/optimization/best-practices/coppa/

Reply:
We agreed to use Google wording for this method. Will change naming for Boolean parameter

Reply:
@TurpIF PM decision was to re-use Google wording yes